### PR TITLE
MINOR: Fix javadoc warnings and define version for storage-common

### PR DESCRIPTION
**Pull request by kkonstantine**
##### Does this solution apply anywhere else?
- [ ] yes
- [ ] no

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [ ] Manual tests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,11 +22,13 @@
...
         <groupId>io.confluent</groupId>
         <artifactId>kafka-connect-storage-common-parent</artifactId>
-        <version>6.0.0-SNAPSHOT</version>
+        <version>10.0.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>kafka-connect-hdfs</artifactId>
+    <version>10.0.0-SNAPSHOT</version>
     <packaging>jar</packaging>
+    <version>10.0.0-SNAPSHOT</version>
```
Comment (levzem) [inline pom.xml:31]: duplicate

File: pom.xml [hunk 2]
```diff
@@ -81,32 +84,32 @@
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-connect-storage-common</artifactId>
-            <version>${confluent.version}</version>
+            <version>${project.version}</version>
```
Comment (levzem) [inline pom.xml:87]: IIUC this means these deps are following the HDFS version, but `kafka-connect-storage-common` isn't guaranteed to be the same version as HDFS, no? shouldn't this be the parent version which is `kafka-connect-storage-common`?

Comment (kkonstantine) [reply pom.xml:87]: they go in lockstep for now. I'll update when I cut the release in storage-common

File: pom.xml (hunk 1 above)
Comment (kkonstantine) [reply pom.xml:31]: Nice catch. I cherry-picked a commit and this made it in.

**Comment by kkonstantine** [comment]
Turns out this had been merged. Changed the title to be more accurate.

**Review by levzem — APPROVED** [review]
LGTM

**Pull request merged**